client/src/router.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Route, Switch } from 'react-router';
+import { BrowserRouter as  Router} from 'react-router-dom';
+import App from './app'
+
+export default () => {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={App}/>
```

Comment:
Can you render App component @ the exact path "/" without any book id number?

Reply:
No, I just can currently render some App components except the data belonging to particular "book id" such as the image, title, author, description and rating details.